MediaBrowser.Providers/Plugins/Omdb/OmdbProvider.cs

```diff
@@ -335,15 +334,25 @@ private async Task<string> EnsureSeasonInfo(string seriesImdbId, int seasonId, C
                     imdbParam,
                     seasonId));
 
-            using var response = await GetOmdbResponse(_httpClientFactory.CreateClient(NamedClient.Default), url, cancellationToken).ConfigureAwait(false);
-            await using var stream = await response.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false);
-            var rootObject = await _jsonSerializer.DeserializeFromStreamAsync<SeasonRootObject>(stream).ConfigureAwait(false);
+            var jsonOptions = JsonDefaults.GetOptions();
+            var rootObject = await GetDeserializedOmdbResponse<SeasonRootObject>(_httpClientFactory.CreateClient(NamedClient.Default), url, jsonOptions, cancellationToken).ConfigureAwait(false);
             Directory.CreateDirectory(Path.GetDirectoryName(path));
-            _jsonSerializer.SerializeToFile(rootObject, path);
+            await using FileStream jsonFileStream = File.OpenWrite(path);
+            await JsonSerializer.SerializeAsync(jsonFileStream, rootObject, jsonOptions, cancellationToken).ConfigureAwait(false);
 
             return path;
         }
 
+        public static async Task<T> GetDeserializedOmdbResponse<T>(HttpClient httpClient, string url, JsonSerializerOptions jsonOptions, CancellationToken cancellationToken)
+        {
+            using var response = await GetOmdbResponse(httpClient, url, cancellationToken).ConfigureAwait(false);
+            var content = await response.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
+
+            // OMDb is sending "N/A" for no empty number fields
+            content = content.Replace("\"N/A\"", "\"\"", StringComparison.InvariantCulture);
```

Comment:
Maybe we should make a custom `JsonConverter` for this?
@crobibero

Reply:
Almost as amazing as tmdb's api!

Reply:
I have an untested converter, if someone could get a sample response I'll wire it up and add tests

Reply:
```
{"Title":"Chapter 1","Year":"2013","Rated":"TV-MA","Released":"01 Feb 2013","Season":"N/A","Episode":"N/A","Runtime":"55 min","Genre":"Drama","Director":"David Fincher","Writer":"Michael Dobbs (based on the novels by), Andrew Davies (based on the mini-series by), Beau Willimon (created for television by), Beau Willimon, Sam Forman (staff writer)","Actors":"Kevin Spacey, Robin Wright, Kate Mara, Corey Stoll","Plot":"Congressman Francis Underwood has been declined the chair for Secretary of State. He's now gathering his own team to plot his revenge. Zoe Barnes, a reporter for the Washington Herald, will do anything to get her big break.","Language":"English","Country":"USA","Awards":"N/A","Poster":"https://m.media-amazon.com/images/M/MV5BMTY5MTU4NDQzNV5BMl5BanBnXkFtZTgwMzk2ODcxMzE@._V1_SX300.jpg","Ratings":[{"Source":"Internet Movie Database","Value":"8.7/10"}],"Metascore":"N/A","imdbRating":"8.7","imdbVotes":"6736","imdbID":"tt2161930","seriesID":"N/A","Type":"episode","Response":"True"}
```
this should be exactly the one that broke it

Reply:
https://github.com/Ullmie02/jellyfin/pull/1

Reply:
I think this can now be updated to not manually remove `N/A`?